rules/windows/execution_suspicious_dotnet_compiler_parent_process.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious dotNet Code execution. Verify process details such as commandline, parent process and network
+connections.
```

Comment:
Can you elaborate more on why this is suspicious

Reply:
This rule aims to detect suspicious execution via dotNet, manifested by suspicious csc.exe or vbc.exe parent process (e.g. svchost.exe spawning csc.exe is almost non occurring in normal conditions and could be a good signal for detecting managed code compilation and execution).

Reply:
Yea that sounds good - I would clean it up a bit and update the `description` with it 👍

Reply:
`.NET`? or dotNET?